MainMenu.java

```diff
@@ -70,38 +70,72 @@ public static void editClientHelper(String input) {
                     Scanner banana = new Scanner(System.in);
                     String newNume = banana.nextLine();
 
-                    //Balanata numerica
-                    if (Objects.equals(parametru, "Balanta")) {
-                        try {Double num = Double.parseDouble(newNume);}
+                    switch (parametru) {
+                        //Balanta numerica
+                        case "Balanta":
+                            try {Double num = Double.parseDouble(newNume);}
                         catch (NumberFormatException e) {
                             System.out.println(color.RED + "\nBalanta trebuie sa fie numerica" + color.RESET);
                             System.exit(0);
                         }
-                    }
+                            break;
 
-                    //Data de facturare 1-31
-                    if (Objects.equals(parametru, "Data de Facturare")) {
-                        try {
+                        //data de facturara  1-31
+                        case "Data de Facturare":
+                            try {
                             int num = Integer.parseInt(newNume);
                         }
                         catch (NumberFormatException e) {
                             System.out.println(color.RED + "\nData trebuie sa fie numerica" + color.RESET);
                         }
                         int date = Integer.parseInt(newNume);
-                        if (1 >= date || date >= 31) System.out.println(color.RED + "\nLuna are 31 zile" + color.RESET);
-                    }
+                        if (1 >= date || date >= 31) System.out.println(color.RED + "\nMonth has 31 days at max" + color.RESET);
+                        break;
 
-                    //date format only
-                    if (Objects.equals(parametru, "Data Nasterii")) {
+                        //date format only
+                        case "Data Nasterii":
                         SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yyyy");
                         try {
-                            Date date = sdf.parse(newNume);
-                            System.out.println("Input date: " + sdf.format(date));
+                            Date date1 = sdf.parse(newNume);
+                            System.out.println("Input date: " + sdf.format(date1));
                         } catch (ParseException e) {
                             System.out.println(color.RED + "\nInvalid date format" + color.RESET);
                             System.exit(0);
                         }
+                        break;
                     }
```

Comment:
aici ai o situatie in care pentru fiecare switch case practic poti sa ai o eroare, de aceea ai folosit un try catch pentru fiecare in parte, ar fi mai bine aranjat daca ai scoate acest try-catch inafara, deci o sa ai switch-ul inauntru la acest try catch, si deja in catch, o sa primesti mai multe erori. aici nu este un caz care ar necesita numaidecat sa faci asta aparte la fiecare